### PR TITLE
LPS-152155 Update context when creating, updating and deleting a default filter

### PR DESCRIPTION
**Pull request by danvb112**
Related Issue: https://issues.liferay.com/browse/LPS-152155

### CODE REVIEW

**Comment by danvb112** [comment]
ci:test:sf

**Comment by danvb112** [comment]
ci:test:relevant

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectView/context.tsx [hunk 1]
```diff
@@ -186,6 +191,71 @@ const viewReducer = (state: TState, action: TAction) => {
...
+				fieldLabel: label[defaultLanguageId],
+				filterBy: label[defaultLanguageId],
+				filterType,
+				label,
+				objectFieldBusinessType,
+				objectFieldName,
+				valueList,
+			};
+
+			// TODO: Update prop that indicates if the column is being used in a filter
```
Comment (naraandrade) [inline modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectView/context.tsx:226]: @danvb112 We forgot to remove this comment.

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectView/index.tsx [hunk 2]
```diff
@@ -38,6 +39,10 @@ const TABS = [
 		Component: DefaultSortScreen,
 		label: Liferay.Language.get('default-sort'),
 	},
+	{
```
Comment (naraandrade) [inline modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectView/index.tsx:42]: @danvb112 We also forgot to remove the tab.

**Review by naraandrade — CHANGES_REQUESTED** [review]

**Comment by danvb112** [comment]
ci:test:sf

**Comment by danvb112** [comment]
ci:test:relevant

**Comment by danvb112** [comment]
ci:test:relevant

**Comment by danvb112** [comment]
ci:forward

**Pull request closed without merging**